File: PYTHON/Kode Program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    __untuk = ""
    __size = ""
    __merk = ""

    def __init__(self, id=0, nama_produk="", harga_produk=0, stok_produk=0, jenis="", bahan="", warna="", untuk="", size="", merk=""):
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk

    # Setter methods
    def set_untuk(self, untuk):
        self.__untuk = untuk

    def get_untuk(self):
        return self.__untuk

    def set_size(self, size):
        self.__size = size

    def get_size(self):
        return self.__size

    def set_merk(self, merk):
        self.__merk = merk

    def get_merk(self):
        return self.__merk

File: JAVA/Kode Program/Baju.java
public class Baju extends Aksesoris {
    private String untuk;   // Untuk jenis hewan apa (Anjing, Kucing, dll.)
    private String size;    // Ukuran baju (S, M, L, XL, dll.)
    private String merk;    // Merk baju

    // Konstruktor default
    public Baju() {
        super();  // Memanggil konstruktor default dari Aksesoris
        this.untuk = "";
        this.size = "";
        this.merk = "";
    }

    // Konstruktor dengan parameter
    public Baju(int id, String nama_produk, double harga_produk, int stok_produk, 
                String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna); // Memanggil konstruktor dari Aksesoris
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    // Setter methods
    public void set_untuk(String untuk) {
        this.untuk = untuk;
    }

    public void set_size(String size) {
        this.size = size;
    }

    public void set_merk(String merk) {
        this.merk = merk;
    }

    // Getter methods
    public String get_untuk() {
        return this.untuk;
    }

    public String get_size() {
        return this.size;
    }

    public String get_merk() {
        return this.merk;
    }

}
